src/components/Tooltip/index.tsx

```diff
@@ -35,10 +39,35 @@ export const Tooltip = ({
   isBoxed = false,
   isTooltipShownOnOverflowOnly = false,
   onCopySuccess,
+  position = TOOLTIP_POSITION.top,
 }: TooltipProps) => {
-  const tooltipChildrenRef = React.useRef<HTMLSpanElement>(null);
+  const tooltipChildrenRef = React.useRef<HTMLElement>(null);
   const tooltipContentRef = React.useRef<HTMLSpanElement>(null);
 
+  const setFixedTooltipPosition = () => {
+    if (!tooltipContentRef.current || !tooltipChildrenRef.current) return;
+    const childElement = tooltipChildrenRef.current as HTMLElement;
+    const tooltipElement = tooltipContentRef.current as HTMLElement;
+    const childBound = childElement.getBoundingClientRect();
+    const tooltipBound = tooltipElement.getBoundingClientRect();
+    switch (position) {
+      case TOOLTIP_POSITION.top: {
+        tooltipContentRef.current.style.top = `${childBound.top - childBound.height - tooltipBound.height / 2}px`;
+        tooltipContentRef.current.style.left = `${childBound.left + childBound.width / 2}px`;
```

Comment:
we already set the left and top in css, why do we add this? what does this do?

Reply:
we set the tooltip position using JS, the one on the CSS is for the arrow(triangle) part of the tooltip